loops-and-arrays.js

```diff
@@ -11,7 +11,11 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // console.log(names) geeft: ["Henk", "Piet", "Fred", "Joop"]
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
-
+names[0] = "Henkje";
+names[1] = "Pietje";
+names[2] = "Fredje";
+names[3] = "Joopje";
+console.log(names);
 
```

Comment:
Dit werkt, maar wanneer de array wordt aangevuld met nog 100 namen wordt dit erg tijdrovend om op deze manier de output te geven. Het is dan makkelijker om gebruik te maken van een for loop:

for (let i = 0; i < names.length; i++) {
  names[i] = names[i] + 'je';
}

console.log(names);